.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: '[BUG]'
+labels: 'type: bug'
+assignees: ''
+---
+
+Thanks for taking the time to file a bug report! Please fill out this form as completely as possible.
```

Comment:
This should be a comment, otherwise it will be added to the issue description.

Reply:
Taken from Rafiki, so it is probably as designed, but I will switch to the web extension templates then.